content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -1,112 +1,243 @@
+This line determines the build farm that executed the tests and the namespace on that cluster in which the execution
+occurred. In this example, the job executed on the `build02` farm and used the `ci-op-2c2tvgti` namespace. All registry
+pullspecs are in the form `<registry>/<namespace>/<imagestream>:<tag>`, so if we needed to access the source image for
+this execution, the pullspec would be `registry.build02.ci.openshift.org/ci-op-2c2tvgti/pipeline:src`. In order to pull
+an image from a test namespace, you must be logged in to the registry and be the author of the pull request. Pull the
+image with any normal container engine:
 
 ```bash
-$ docker pull registry.build02.ci.openshift.org/ci-op-2c2tvgti/pipeline:odh-manifests-tests
+$ podman pull registry.build02.ci.openshift.org/ci-op-2c2tvgti/pipeline:src
 ```
 
-Note that the only github user that has access to the image is the author of the underlying pull request which triggers the testrun.
+**Warning:** Only `vSphere` system administrators can access the images on [registry.apps.build01-us-west-2.vmc.ci.openshift.org](registry.apps.build01-us-west-2.vmc.ci.openshift.org).
+
+## How do I access the latest published images for my component?
 
-# Access the promoted images
+If the `ci-operator` configuration for your component configures image [`promotion`](/docs/architecture/ci-operator/#publishing-container-images),
+output container images will be published to the central CI registry when changes are merged to your repository. Two main
+configurations are possible for promotion: configuring an `ImageStream` name or a tag.
 
-If `ci-operator`'s config has [`promotion` stanza](/docs/architecture/ci-operator/#publishing-container-images), the images are published to the central CI registry at `app.ci`.
+### Publication of New Tags
+
+A configuration that specifies the `ImageStream` name looks like the following and results in new tags on that stream:
 
 ```yaml
 images:
-- dockerfile_path: images/hello-openshift/Dockerfile.rhel
+- dockerfile_path: images/my-component
   from: base
-  inputs:
-    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
-      as:
-      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
-  to: hello-openshift
+  to: my-component
 promotion:
   name: "4.7"
   namespace: ocp
 ```
 
-We can pull the image defined by above `promotion` stanza by:
+The `my-component` image can be pulled from the authoritative registry with:
 
 ```bash
-$ docker pull registry.ci.openshift.org/ocp/4.7:hello-openshift
+$ podman pull registry.ci.openshift.org/ocp/4.7:my-component
 ```
 
-# Troubleshooting
+### Publication of New Streams
+
+A configuration that specifies the `ImageStream` tag looks like the following and results in new streams in the namespace:
+
+```yaml
+images:
+- dockerfile_path: images/my-component
+  from: base
+  to: my-component
+promotion:
+  namespace: my-organization
+  tag: latest
```

Comment:
this is very weird.  what happens if i have multiple images defined?  i get a new imagestream for each one but they all use the same tag name within their respective imagestreams?

Reply:
Yes

Reply:
can you make this behavior clear in the docs?

Reply:
nm, looks like you did